src/constants.js

```diff
@@ -28,6 +28,10 @@ const CONTRACT_ADDRESSES = {
   [ContractNames.WithdrawalDelayer]: '0x8EEaea23686c319133a7cC110b840d1591d9AeE0'
 }
 
+const STORAGE_VERSION_KEY = 'version'
```

Comment:
Add hermezjs prefix as the wallet will have its own version key and hermezjs can be used in other projects